localstack/testing/pytest/fixtures.py

```diff
@@ -724,6 +724,32 @@ def _create_key(**kwargs):
             LOG.debug("error cleaning up KMS key %s: %s", key_id, e)
 
 
+# kms_create_key fixture is used here not just to be able to create aliases without a key specified,
+# but also to make sure that kms_create_key gets executed before and teared down after kms_create_alias -
+# to make sure that we clean up aliases before keys get cleaned up.
+@pytest.fixture()
+def kms_create_alias(kms_client, kms_create_key):
+    aliases = []
+
+    def _create_alias(**kwargs):
+        if "AliasName" not in kwargs:
+            kwargs["AliasName"] = f"alias/{short_uid()}"
+        if "TargetKeyId" not in kwargs:
+            kwargs["TargetKeyId"] = kms_create_key()["KeyId"]
+
+        kms_client.create_alias(**kwargs)
+        aliases.append(kwargs["AliasName"])
+        return kwargs["AliasName"]
```

Comment:
```suggestion
        aliases.append(kwargs["AliasName"])
        return kms_client.create_alias(**kwargs)
```

returning the whole response is more flexible and is then a "proper" replacement for something like `kms_client.create_alias(...)`

Reply:
There is no response, unfortunately. According to the docs, create_alias returns nothing. So I return the name of an auto-generated alias even if it doesn't comply with the AWS API, as otherwise getting to that name would be burdensome.